src/read/read_ref.rs

```diff
@@ -65,23 +62,34 @@ where
     /// Note that not all writes will be included with this read -- only those that have been
     /// refreshed by the writer. If no refresh has happened, or the map has been destroyed, this
     /// function returns `None`.
-    pub fn get<'a, Q: ?Sized>(&'a self, key: &'_ Q) -> Option<&'a Values<V, S>>
+    pub fn get<Q: ?Sized>(&'rh self, key: &'_ Q) -> Option<&'rh Values<V, S>>
```

Comment:
Looking at this (and `get_one`) again, I don't _think_ this should be tied to `'rh`. I mean, it _can_ be, but what matters is that the return value is tied to the lifetime of `&self`. This updated form requires that the reference to the handle live as long as the `ReadGuard`'s borrow, which isn't a necessary requirement. This should therefore probably stay the way it was, as that is the most relaxed bound that is still safe. And `get_one` should have the same.